swatch-producer-aws/build.gradle

```diff
@@ -7,52 +7,44 @@ plugins {
 }
 
 dependencies {
+    compileOnly libraries["lombok"]
```

Comment:
almost all of these dependency changes were just reordering/alphabetizing.  then the addition of the splunk logging dependency